doc/api/zip.md

```diff
@@ -0,0 +1,50 @@
+# Zip
+
+<!--introduced_in=REPLACEME-->
+
+> Stability: 1 - Experimental
+
+<!-- source_link=lib/libzip.js -->
+
+The `node:zip` module provides tools allowing to manipulate zip archives.
+
+To access it:
+
+```js
+const zip = require('node:zip');
+```
+
+Adding files to an archive may be done using the `addEntry` family of
+functions:
+
+```js
+const { ZipArchive } = require('node:zlib');
```

Comment:
Should this be `require('node:zip')` instead?

Reply:
Yes, typo (although there are small debates on whether it should live in `node:zip` / `node:archive`, or reuse `node:zlib`; cf https://github.com/nodejs/node/pull/45651#issuecomment-1330718544 and https://github.com/nodejs/node/pull/45651#issuecomment-1330761223)